{"name": "test_six[6-9-13]", "status": "failed", "statusDetails": {"message": "AssertionError: assert 15 == 13\n +  where 15 = add(6, 9)", "trace": "self = <Work.test_work.TestB object at 0x10b53af60>, a = 6, b = 9, expected = 13\n\n    @pytest.mark.parametrize(\"a,b,expected\", json.load(open(\"calc.json\")))\n    def test_six(self, a, b, expected):\n        logging.info(\"This is test_six!!!\")\n>       assert add(a, b) == expected\nE       assert 15 == 13\nE        +  where 15 = add(6, 9)\n\ntest_work.py:58: AssertionError"}, "attachments": [{"name": "log", "source": "16878194-c15f-438c-8f05-758c2d71c6ea-attachment.txt", "type": "text/plain"}, {"name": "stderr", "source": "66dcf33c-a672-488d-b83d-5260d2d7361a-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "a", "value": "6"}, {"name": "b", "value": "9"}, {"name": "expected", "value": "13"}], "start": 1572058204827, "stop": 1572058204828, "uuid": "02451de4-3e24-4fd1-80c4-20da76a55520", "historyId": "fe82b554876d75cd5a307d34c53dd79f", "testCaseId": "33cda52eda96a5e8528fcc03f1cd347a", "fullName": "test_work.TestB#test_six", "labels": [{"name": "suite", "value": "test_work"}, {"name": "subSuite", "value": "TestB"}, {"name": "host", "value": "ParsondeMacBook-Pro.local"}, {"name": "thread", "value": "6564-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_work"}]}